### DISCUSSION TITLE
Configuring Maximum Duration for Hobby on Vercel Functions in Django

### DISCUSSION THREAD

**Discussion by livillivil** [vercel/community#7094] [Help]
### Summary

Based on this changelog:  https://vercel.com/changelog/vercel-functions-for-hobby-can-now-run-up-to-60-seconds
As per new changelog Now hubby accounts on vercel can increase limit up to 60 seconds. My Django app works perfectly for this vercel.json::

{
"version": 2,
"builds": [{"src": "omninepse/wsgi.py","use": "@vercel/python","config": { "maxLambdaSize": "15mb", "runtime": "python3.12" }}],
"routes": [{"src": "/(.*)","dest": "omninepse/wsgi.py"}]
}

To increase the limit allowed upto 50sec for hubby acc I changed as below which failed to deploy::

{
"version": 2,
"builds": [{"src": "omninepse/wsgi.py","use": "@vercel/python","config": { "maxLambdaSize": "15mb", "runtime": "python3.12" }}],
"routes": [{"src": "/(.*)","dest": "omninepse/wsgi.py"}],
"functions": {"stocks/ofun.py": {"maxDuration": 50}}
}

### Steps to Reproduce

{
"version": 2,
"builds": [{"src": "omninepse/wsgi.py","use": "@vercel/python","config": { "maxLambdaSize": "15mb", "runtime": "python3.12" }}],
"routes": [{"src": "/(.*)","dest": "omninepse/wsgi.py"}],
"functions": {"stocks/ofun.py": {"maxDuration": 50}}
}

**Comment by abhiamishra** [comment]
I am also encountering this same issue with a FastAPI server. 

When I have this block:
```
"functions": {
    "api/*.py": {
      "maxDuration": 60
    }
  }
```
it does not seem to work and Vercel deployment fails.

Requesting help please

**Comment by zakutnya** [comment]
I do have this problem too. My plan is Hobby, i'm trying to increase `maxDuration` of the only function in my Django project (`api/wsgi.py`), and when i add this piece of json to `vercel.json` the deployment doesn't even start. As soon as i remove it, everything's back working again.

```json
{
    "version": 2,
    "builds": [
        {
            "src": "api/wsgi.py",
            "use": "@vercel/python",
            "config": { "maxLambdaSize": "15mb", "runtime": "python3.12" }
        },
        {
            "src": "build_frontend.sh",
            "use": "@vercel/static-build",
            "config": {
                "distDir": "static/client/dist"
            }
        },
        {
            "src": "build_backend.sh",
            "use": "@vercel/static-build",
            "config": {
                "distDir": "static"
            }
        }
    ],
    "routes": [
        {
            "src": "/vendor/assets/(.*)",
            "dest": "/assets/$1"
        },
        {
            "src": "/vendor/(.*)",
            "dest": "/static/$1"
        },
        {
            "src": "/(.*)",
            "dest": "api/wsgi.py"
        }
    ],
    // ↓ if i put this here the build process doesn't even start 
    "functions": {
        "api/wsgi.py": {
            "maxDuration": 60
        }
    }
}
```

<img width="1259" alt="image" src="https://github.com/vercel/community/assets/1798464/0c3998bf-bec4-4e14-8e32-714aadb07685">

The project and the final output structure is formed according to the docs:
<img width="425" alt="image" src="https://github.com/vercel/community/assets/1798464/02142021-9756-436f-a494-365d2d5788cb">

**Reply by abhiamishra** [reply]
Exact same issue - its frustrating because on the website it says this is now in nextJS so I wonder why it just fails

**Comment by Darvinbond** [comment]
Exact same problem in facing. Any fixes yet?

[a comment by an automated account is not shown]
